api/src/main/java/gov/cms/ab2d/api/controller/StatusAPI.java

```diff
@@ -0,0 +1,191 @@
+package gov.cms.ab2d.api.controller;
+
+import gov.cms.ab2d.api.config.SwaggerConfig;
+import gov.cms.ab2d.common.model.Job;
+import gov.cms.ab2d.common.model.JobOutput;
+import gov.cms.ab2d.common.service.JobService;
+import io.swagger.annotations.*;
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.dstu3.model.DateTimeType;
+import org.slf4j.MDC;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.constraints.NotBlank;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static gov.cms.ab2d.api.controller.BulkDataAccessAPI.JOB_CANCELLED_MSG;
+import static gov.cms.ab2d.api.controller.BulkDataAccessAPI.JOB_NOT_FOUND_ERROR_MSG;
+import static gov.cms.ab2d.api.util.Constants.GENERIC_FHIR_ERR_MSG;
+import static gov.cms.ab2d.api.util.SwaggerConstants.BULK_CANCEL;
+import static gov.cms.ab2d.common.service.JobService.ZIPFORMAT;
+import static gov.cms.ab2d.common.util.Constants.*;
+
+@Slf4j
+@Api(value = "Bulk Data Access API", description = "API to determine the status of the job, the files to download " +
+        "once the job is complete and an endpoint to cancel a job",
+        tags = {"Status"})
```

Comment:
Since this has delete in it too, can we add a tag for Delete?

Reply:
Both GET and DELETE for /status are under this. Technically delete is status.

Reply:
Oh, and in case it was clear from the code, the "tags" item is what defines the title of the section.